<launch>
    <arg name="keypoint_detect"              default="False" />
    <arg name="sub_image_raw_topic_name"     default="/camera/rgb/image_raw" />
    <arg name="pub_face_detected_topic_name" default="/kamerider_image_detection/face_detect_result" />
    <arg name="path_to_pretrained_dataset"   default="$(find kamerider_image_detection)/shape_predictor_68_face_landmarks.dat" />
    <arg name="path_to_save_image"           default="$(find kamerider_image_detection)/face_detection_result.png" />
    
    <node name="face_detection" pkg="kamerider_image_detection" type="face_detection.py" output="screen">
        <param name="keypoint_detect"              type="bool"   value="$(arg keypoint_detect)" />
        <param name="sub_image_raw_topic_name"     type="string" value="$(arg sub_image_raw_topic_name)" />
        <param name="pub_face_detected_topic_name" type="string" value="$(arg pub_face_detected_topic_name)" />
        <param name="path_to_pretrained_dataset"   type="string" value="$(arg path_to_pretrained_dataset)" />
        <param name="path_to_save_image"           type="string" value="$(arg path_to_save_image)" />
    </node>
</launch>
